charts/descheduler/values.yaml

```diff
@@ -79,6 +79,11 @@ leaderElection: {}
 #  resourceName: "descheduler"
 #  resourceNamespace: "kube-system"
 
+metricsCollector:
```

Comment:
I assume it's difficult to iterate through profiles and infer if we need to enable this RBAC permission?

Reply:
sorry but I don't get what you mean.

Reply:
In other words, why can't we use the following flag:
`deschedulerPolicy.metricsCollector.enabled`

```
{{- if and .Values.deschedulerPolicy.metricsCollector .Values.deschedulerPolicy.metricsCollectorenabled }}
```

Reply:
indeed, that seems way more reasonable